test/integration/distributed-tracing/traceparent-sampled-flag.test.js

```diff
@@ -292,38 +290,6 @@ test('remote_parent_sampled: default, remote_parent_not_sampled: default (flag f
   }
 })
 
-test('traceparent: no sample, tracestate: sample', async t => {
```

Comment:
isn't this test still valid? it's asserting that the traceparent is not sampled so it should set the sampled/priority from trace state